solr/core/src/java/org/apache/solr/cli/PackageTool.java

```diff
@@ -310,6 +310,18 @@ private Pair<String, String> parsePackageVersion(String arg) {
   public List<Option> getOptions() {
     return List.of(
         SolrCLI.OPTION_SOLRURL,
+        // Cannot use the default OPTION_SOLRPORT, because -p is already taken by PARAMS
```

Comment:
Yeah...   I wonder if we should make the slightly breaking chagne?   This feels like we are going to get bit by this in the future?  Or, do we think humans will never actually use teh `-p` parameter to mean a solr port, becuase it's a config file type thing...   I could see just forcing --package, cause how often do you type this command?  whereas -p is super common IF we think humans will use these parameters a lot.  <-- did that make sense?